js/views/modals/purchase/Purchase.js

```diff
@@ -322,13 +323,18 @@ export default class extends BaseModal {
 
     this.setState({ phase: 'processing' });
 
+    const segmentation = { error: 'None' };
```

Comment:
This is another place we're not so consistent. In most place success is defined as an empty error string, whereas here we went with 'None'.